heron/api/tests/java/com/twitter/heron/streamlet/impl/StreamletImplTest.java

```diff
@@ -262,9 +263,9 @@ public void testComplexBuild() throws Exception {
 
   @Test
   public void testResourcesBuilder() {
-    Resources defaultResoures = Resources.defaultResources();
-    assertEquals(0, Float.compare(defaultResoures.getCpu(), 1.0f));
-    assertEquals(defaultResoures.getRam(), ByteAmount.fromMegabytes(100));
+    Resources defaultResources = Resources.defaultResources();
+    assertEquals(0, Float.compare(defaultResources.getCpu(), 1.0f));
+    assertEquals(defaultResources.getRam(), 104857600);
```

Comment:
@erenavsarogullari - ByteAmount is a convenience class that we use to make it more readable. Any idea why we need to change ByteAmount?